packages/apputils/src/vdom.ts

```diff
@@ -107,6 +107,18 @@ abstract class VDomRenderer<T extends VDomRenderer.IModel | null> extends Widget
 }
 
 
+export
+class ElementVDomRenderer extends VDomRenderer<any> {
```

Comment:
Not sure that the `ElementVDomRenderer` name quite captures the purpose of this class. Don't have better ideas right now though. Thoughts?

Reply:
I don't like the name either. What about `ReactWidget`?

Reply:
That works for me.

Reply:
Updated.